pilot/cmd/pilot-agent/main.go

```diff
@@ -245,7 +248,15 @@ var (
 				opts := make(map[string]string)
 				opts["PodName"] = os.Getenv("POD_NAME")
 				opts["PodNamespace"] = os.Getenv("POD_NAMESPACE")
-				opts["MixerSubjectAltName"] = envoy.GetMixerSAN(opts["PodNamespace"])
+
+				if role.MixerIdentity == "" {
+					opts["MixerSubjectAltName"] = envoy.GetSAN(opts["PodNamespace"], envoy.MixerSvcAccName)
+				} else {
+					opts["MixerSubjectAltName"] = envoy.GetSAN("", role.MixerIdentity)
+				}
+
+				//override value for pilot of call mixed if custom SAN/SPIFFE is being used
+				opts[".MixerSpiffeOverride"] = os.Getenv("MIXER_SPIFFE_OVERRIDE")
```

Comment:
This part confused me. Why do we need to have MIXER_SPIFFE_OVERRIDE here?

Reply:
it is not needed I delete the line